lib/worldwide/rails/railtie.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "rails/railtie"
+
+module Worldwide
+  module Extensions
```

Comment:
Should this be included in the file structure too? Usually we just put the railtie under the main module to make it easier to find.

Reply:
Oops the `Extension` module is no longer a thing. I'll take it out.